frontend/src/components/Export.js

```diff
@@ -72,6 +113,32 @@ class Export extends Component {
               >
                 <option value="Davies">Davies Student Center</option>
                 <option value="Library">McIntyre Library</option>
+                <option value="Bridgman">Bridgman</option>
```

Comment:
These really should be created from an API call. Not hard-coded.

Reply:
For the buildings, I think I am still waiting on a table with all the current building names if there isn't one already.